lindera-unidic-builder/README.md

```diff
@@ -74,11 +122,30 @@ You can tokenize text using produced dictionary with `lindera` command:
 ```
 
 ```text
-羽田    名詞,固有名詞,人名,姓,*,*,羽田,ハタ,ハタ
-空港    名詞,普通名詞,一般,*,*,*,空港,クーコー,クーコー
-限定    名詞,普通名詞,サ変可能,*,*,*,限定,ゲンテー,ゲンテー
-トート  名詞,普通名詞,一般,*,*,*,トート,トート,トート
-バッグ  名詞,普通名詞,一般,*,*,*,バッグ,バッグ,バッグ
+羽田    名詞,固有名詞,人名,姓,*,*,ハタ,ハタ,羽田,ハタ,羽田,ハタ,固,*,*,*,*
+空港    名詞,普通名詞,一般,*,*,*,クウコウ,空港,空港,クーコー,空港,クーコー,漢,*,*,*,*
+限定    名詞,普通名詞,サ変可能,*,*,*,ゲンテイ,限定,限定,ゲンテー,限定,ゲンテー,漢,*,*,*,*
+トート  名詞,普通名詞,一般,*,*,*,トート,トート,トート,トート,トート,トート,外,*,*,*,*
+バッグ  名詞,普通名詞,一般,*,*,*,バッグ,バッグ-bag,バッグ,バッグ,バッグ,バッグ,外,*,*,*,*
+EOS
+```
+
+## Tokenizing text using default dictionary and produced binary user dictionary
+
+You can tokenize text using produced dictionary with `lindera` command:
+
+```shell script
+% echo "東京スカイツリーの最寄り駅はとうきょうスカイツリー駅です" | lindera -k unidic -u ./resources/unidic_userdic.bin -t binary
```

Comment:
What's happen if we provide `-k unidic -u ipadic_userdic.bin`?

Reply:
Basically, it works.
However, notice that the detailed information on the words will be IPADIC's.

```
$ echo "東京スカイツリーの最寄り駅はとうきょうスカイツリー駅です" | ./target/debug/lindera -k unidic -u ./resources/ipadic_userdic.bin -t binary
東京スカイツリー        カスタム名詞,*,*,*,*,*,東京スカイツリー,トウキョウスカイツリー,*
の      助詞,格助詞,*,*,*,*,ノ,の,の,ノ,の,ノ,和,*,*,*,*
最寄り  名詞,普通名詞,一般,*,*,*,モヨリ,最寄り,最寄り,モヨリ,最寄り,モヨリ,和,*,*,*,*
駅      名詞,普通名詞,一般,*,*,*,エキ,駅,駅,エキ,駅,エキ,漢,*,*,*,*
は      助詞,係助詞,*,*,*,*,ハ,は,は,ワ,は,ワ,和,*,*,*,*
とうきょうスカイツリー駅        カスタム名詞,*,*,*,*,*,とうきょうスカイツリー駅,トウキョウスカイツリーエキ,*
です    助動詞,*,*,*,助動詞-デス,終止形-一般,デス,です,です,デス,です,デス,和,*,*,*,*
EOS
```

Reply:
If possible, we mention this behaviour and there is no error.What do you think?